builds/e2e/longhaul.yaml

```diff
@@ -92,6 +94,10 @@ jobs:
           longHaul.desiredModulesToRestartCSV: '$(longHaul.desiredModulesToRestartCSV)'
           longHaul.restartIntervalInMins: '$(longHaul.restartIntervalInMins)'
           analyzer.consumerGroupId: 'longhaul_linux_amd64'
+          analyzer.loganalyticenabled: '$(logAnalytic.enabled)'
+          analyzer.loganalyticworkspaceid: '$(LogAnalyticWorkspaceId)'
+          analyzer.loganalyticsharedkey: '$(LogAnalyticSharedKey)'
+          analyzer.loganalyticlogtype: '$(logAnalytic.logType)'
```

Comment:
are you going to have different logType for different OS/architecture?

Reply:
No. Per discussiong, the OS/Arch/OsBit are going to be just another field in the message. 
The LogType will be used for a different type of testing : Stress, LH, E2E.